# Generated by Django 5.2.6 on 2025-09-06 05:50

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('logs', '0005_databasepermission'),
    ]

    operations = [
        migrations.AddField(
            model_name='customuser',
            name='failed_login_attempts',
            field=models.PositiveIntegerField(default=0, help_text='Số lần đăng nhập thất bại liên tiếp', verbose_name='Số lần đăng nhập thất bại'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='last_failed_login',
            field=models.DateTimeField(blank=True, help_text='Thời gian đăng nhập thất bại cuối cùng', null=True, verbose_name='Lần đăng nhập thất bại cuối'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='locked_until',
            field=models.DateTimeField(blank=True, help_text='Thời gian tài khoản bị khóa đến (None = không bị khóa)', null=True, verbose_name='Khóa đến'),
        ),
    ]
